frontend/admin/src/js/components/home/HomePage.test.tsx

```diff
@@ -2,25 +2,12 @@
  * @jest-environment jsdom
  */
 import "@testing-library/jest-dom";
-import { render } from "@testing-library/react";
 import { axe } from "jest-axe";
 import React from "react";
-import { IntlProvider, MessageFormatElement } from "react-intl";
+import { render } from "@common/helpers/testUtils";
 import HomePage from "./HomePage";
 
-const renderWithReactIntl = (
-  component: React.ReactNode,
-  locale?: "en" | "fr",
-  messages?: Record<string, string> | Record<string, MessageFormatElement[]>,
-) => {
-  return render(
-    <IntlProvider locale={locale || "en"} messages={messages}>
-      {component}
-    </IntlProvider>,
-  );
-};
-
-const renderHomePage = () => renderWithReactIntl(<HomePage />);
+const renderHomePage = () => render(<HomePage />);
```

Comment:
render is the default name, so it's it's rendered with other things, seems worth it not to hide that? even just renderWithProviders gives a hint at what happens below the abstraction in another file, instead of obfuscating and requiring opening it up :)

Reply:
sounds good, will mark this as resolved for now, and reference below as "to be re-ticketed" list :)